index.html

```diff
@@ -2808,129 +2830,254 @@ <h2 id="docconformance">
+          add accessibility information to HTML elements using the Accessible
+          Rich Internet Applications specification (ARIA 1.1).
+          </li>
+          <li>[[[wai-aria-practices-1.2]]] - An author's guide to understanding
+          and implementing Accessible Rich Internet Applications.
+          </li>
+        </ul>
+      </div>
+      <aside class="example">
+        <p>
+          These features can be used to make accessibility tools render content
+          to their users in more useful ways. For example, ASCII art, which is
+          really an image, appears to be text, and in the absence of
+          appropriate roles and properties would end up being rendered by
+          screen readers as a nonsensical string of punctuation characters.
+          Using the features described in this section, one can instead make
+          the ATs skip the ASCII art and just read the caption:
+        </p>
+        <pre>
+&lt;figure&gt;
+  &lt;pre role="img"&gt;
+  o           .'`/
+    '      /  (
+  O    .-'` ` `'-._      .')
+      _/ (o)        '.  .' /
+      )       )))     &gt;&lt;  &lt;
+      `\  |_\      _.'  '. \
+        '-._  _ .-'       '.)
+    jgs     `\__\
+  &lt;/pre&gt;
+  &lt;figcaption <strong>id="fish-caption"</strong>&gt;
+    Joan G. Stark, "&lt;cite&gt;fish&lt;/cite&gt;".
+    October 1997. ASCII on electrons. 28×8.
+  &lt;/figcaption&gt;
+&lt;/figure&gt;
+</pre><!-- source: http://www.geocities.com/SoHo/7373/aquatic.htm#fish -->
+      </aside>
+
+      <h3 id="docconformance-attr">
+        Document conformance requirements for use of ARIA attributes with HTML elements with HTML attribute parity
+      </h3>
+      <p>
+        Unless otherwise stated, authors MAY use `aria-*` attributes in place of their HTML equivalents on HTML elements where the `aria-*` semantics would be expected. For example, authors MAY use `aria-disabled=true` on a `button` rather than the `disabled` attribute. However, authors SHOULD NOT use both the native HTML attribute and the `aria-*` attribute together. As stated in <a data-cite="wai-aria-1.1#host_general_conflict">WAI-ARIA's Conflicts with Host Language Semantics</a>, user agents MUST ignore WAI-ARIA attributes and use the host language (HTML) attribute with the same implicit semantics.
+      </p>
+      <p>
+        The following table represents HTML elements and their attributes which have `aria-*` attribute parity.
+      </p>
+      <table class="simple">
+        <caption>
+          Rules of ARIA attribute usage by HTML elements
+        </caption>
+        <thead>
+          <tr>
+            <th>
+              HTML feature
+            </th>
+            <th>
+              <p id="implicit-attr">
+                Implicit ARIA semantics
+              </p>
+            </th>
+            <th>
+              Author guidance
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr id="att-checked" tabindex="-1">
+            <th>
```

Comment:
while semantically correct to use header cells for each row, the current standard styling makes this table look...rather unappealing. and it doesn't now match the structure of the other tables. might be worth just reverting to `<td>`s? or tweaking the styles and also modifying the other tables?

Reply:
I made changes to a css file to modify the styling, but that isn’t being rendered as part of the preview

Reply:
ah, gotcha. but yeah, then, changing all other tables to match? can probably be done with a semi-clever find/replace?

Reply:
Yup. Was going to do that after this pr since that’d show “changes” everywhere then.